draft-ietf-tls-tls13.md

```diff
@@ -2939,6 +2939,12 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+Servers are permitted to buffer and postpone the processing of application
+data sent in 0-RTT until they receive client's Finished. Therefore, when
+using 0-RTT, clients MUST proceed the handshake to the point of sending
+their Finished messages even if there are no application data to be sent
```

Comment:
nit: Has plural for clients, but singular for handshake, thus Finished messages should be singular, nor plural, for consistency. (one handshake has one Finished message from the client) So, instead of "their Finished messages", do "the Finished message". (also "the" instead of "their to match use in prior paragraph)